terraform/telemetry.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# File usage:
+# python3 telemetry.py session project event version
```

Comment:
This header makes me think those are all positional arguments, but they're not. Maybe show example usage?

`python3 telemetry.py --session 123  --project my-project --event my-event --version v0.0.0`